com.unity.mobile.notifications/Runtime/Android/AndroidNotificationCenter.cs

```diff
@@ -48,10 +48,16 @@ public class AndroidNotificationCenter
 
         private static AndroidJavaClass s_NotificationManagerClass;
         private static AndroidJavaObject s_NotificationManager;
-        private static AndroidJavaObject s_NotificationManagerContext;
         private static AndroidJavaObject s_CurrentActivity;
         private static bool s_Initialized;
 
+        private static AndroidJavaObject Notification_EXTRA_TITLE;
+        private static AndroidJavaObject Notification_EXTRA_TEXT;
+        private static AndroidJavaObject Notification_EXTRA_SHOW_CHRONOMETER;
+        private static AndroidJavaObject Notification_EXTRA_BIG_TEXT;
+        private static int Notification_FLAG_AUTO_CANCEL;
+        private static int Notification_FLAG_GROUP_SUMMARY;
```

Comment:
What are these used for?

Reply:
They are used to convert Notification Java object back to C# counterpart.